pyproject.toml

```diff
@@ -511,13 +511,14 @@ filterwarnings = [
 junit_family = "xunit2"
 markers = [
     "all_backends: mark tests that run against all execution engine backends",
+    "athena: mark tests that require pyathena",
     "aws_creds: mark all tests that require aws credentials",
     "big: mark tests that don't have external dependencies but aren't unit tests",
     "cli: mark all tests in the /cli directory.",
     "cloud: mark test as being relevant to Great Expectations Cloud.",
     "docs: mark a test as a docs test.",
     "e2e: mark test as an E2E test.",
-    "external_sqldialect: mark test as requiring install of an external sql dialect.",
+    "google_creds: mark tests that require google creds",
```

Comment:
🚀

Reply:
Oh, I can't actually remove `"external_sqldialect`. We don't use it for our test markers but another pipeline uses it and I've left the marker in the tests.